### PR TITLE
[Bocaj] Third(-ish) Annual Refactor and Addition of PlanckEZ Layout

### PR DESCRIPTION
**Pull request by JacobJerrell**
## Description

Slimmed things down quite a bit, and then added quite a few things for the Planck

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [ ] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: layouts/community/ergodox/bocaj/config.h [hunk 1]
```diff
@@ -1,6 +1,6 @@
 #pragma once
-
 #include QMK_KEYBOARD_CONFIG_H
```
Comment (drashna) [inline layouts/community/ergodox/bocaj/config.h:2]:
```suggestion
```

File: layouts/community/ergodox/bocaj/config.h [hunk 2]
```diff
@@ -1,6 +1,6 @@
...
 
 #define LEADER_TIMEOUT 250
 #define LEADER_PER_KEY_TIMING
+#undef TAP_DANCE_ENABLE
```
Comment (drashna) [inline layouts/community/ergodox/bocaj/config.h:6]: This isn't were to set this.

You'd want to use the rules.mk to disable tapdance.

Comment (JacobJerrell) [reply layouts/community/ergodox/bocaj/config.h]: Thank you, it had already been set in rules.mk. Recent commit has removed this line

**Comment by drashna** [comment]
Could you revert the changes to `lib/vusb`?   If needed, run `make git-submodule` and commit the changes.  That may fix the issue.

**Pull request closed without merging**